fastapi_sqla/aws_rds_iam_support.py

```diff
@@ -0,0 +1,30 @@
+import boto3
```

Comment:
Since `boto3` is optional, this will fail to import on systems that don't have it. We have to import it in a try/except.

I suggest that when boto3 is not set we also raise on [this line](https://github.com/dialoguemd/fastapi-sqla/pull/43/files#diff-b2ab2a53b6ed37e0cc84a6d09670f9e1fb169ba79fb25898d06ab9ce56776415R10) with a message explaining that boto3 must be installed.

Reply:
[This approach looks good to me](https://github.com/dialoguemd/fastapi-sqla/blob/master/fastapi_sqla/__init__.py#L30)

Reply:
Done